bitnami/mongodb/templates/arbiter/statefulset.yaml

```
@@ -271,6 +281,12 @@ spec:
             - key: mongodb-ca-key
               path: mongodb-ca-key
               mode: 0600
+        {{- else }}
+        - name: mongodb-certs-0
+          secret:
+            secretName: {{ .Values.tls.arbiter.existingSecret }}
```

If we open the possibility to evaluate entries in `.Values.tls.X.existingSecrets` paths as templates, I think we should also do it here. WDYT?

```suggestion
            secretName: {{ include "common.tplvalues.render" ( dict "value" .Values.tls.arbiter.existingSecret "context" $) }}
```

---

Agree too, changed :)